src/lightning_app/components/multi_node/pytorch_spawn.py

```diff
@@ -31,21 +33,37 @@ def __call__(
     ):
         import torch
 
+        # Remove the wrapper.
+        setattr_fn = self.work._setattr_replacement
```

Comment:
let's add a method that removes the wrapper (and another that restores it), in case we need to do it in other places + the operations you need to do to remove the wrapper will change

It could actually be a context
```
with self.no_wrapper():
    ...
```
your call

Reply:
Done !